elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -3,252 +3,379 @@
 $LOCAL: tabs;
 
 $LOCAL-VARIABLES: (
-  BackgroundColor: transparent,
-  Color:           pfe-var(ui-disabled--text),
-  link:            pfe-var(link),
-  focus:           pfe-var(link--focus),
-  highlight:       pfe-var(ui-accent),
-  tab: (
-    FontSize:       pfe-var(font-size),
-    PaddingTop:     calc(#{pfe-var(container-padding)} * .666),
-    PaddingRight:   calc(#{pfe-var(container-padding)} * 1.5),
-    PaddingBottom:  calc(#{pfe-var(container-padding)} * .666),
-    PaddingLeft:    calc(#{pfe-var(container-padding)} * 1.5),
+  BackgroundColor:           transparent,
+  BackgroundColor--inactive: pfe-var(surface--lighter),
+  Color:                     pfe-var(text--muted),
+  Color--focus:              #151515,
+  BorderColor:               pfe-var(ui-accent),      // active is bright
+  BorderColor--hover:        pfe-var(surface--base),  // hover is gray
+  BorderWidth:               pfe-var(ui--border-width--active),
+
+  tab: ( // to avoid collisions with panel
+    FontSize:                pfe-var(font-size),
+    Margin:                    0 0 calc( #{pfe-var(ui--border-width)} * -1),
+    PaddingTop:              pfe-var(container-padding),
+    PaddingRight:            calc(#{pfe-var(container-padding)} * 2),
+    PaddingBottom:           pfe-var(container-padding),
+    PaddingLeft:             calc(#{pfe-var(container-padding)} * 2),
   )
 );
 
+// TODO: These will be deprecated at 1.0  
+$backwards-compatibility: (
```

Comment:
```
$backwards-compatibility: (
  Color--focus:  pfe-local(focus, #151515), 
  BorderColor:  pfe-local(highlight, pfe-var(ui-accent)),
);
```

Ah, thank you for the comment re: highlight because that reminded me that the backwards compatibility map should look like this instead.

Reply:
updated